### PR TITLE
New Keyboard layout English Spanish Catalan

### PR DESCRIPTION
**Pull request by Norgus**
I've added a layout based on English thumbkey layout supporting Spanish & Catalan. Inspiration drawn from ES-CA thumbkey layout.

I think I've done it right, but it's  fairly unfamiliar stuff to me.
Reason for this layout is I already 'mapped' the English layout in my head but want to be able to type the Catalan and Spanish accents without stumbling.

### CODE REVIEW

File: app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt [hunk 1]
```diff
@@ -234,4 +235,5 @@ enum class KeyboardLayout(val keyboardDefinition: KeyboardDefinition) {
     ENHRMessagEase(KB_EN_HR_MESSAGEASE),
     CZprogrammerMessagease(KB_CZ_PROG),
     ENDEThumbKeyV2(KB_EN_DE_THUMBKEY_V2),
+    ENESCA_THUMBKEY(KB_EN_ES_CA_THUMBKEY),
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt:238]: Make sure this one is named like the others, no underscore.

**Review by dessalines — COMMENTED** [review]
Run `./gradlew formatKotlin` to pass lint.

**Comment by Norgus** [comment]
> Run `./gradlew formatKotlin` to pass lint.

I don't know what I'm doing with this, but I tried with the following failure.

File: app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt (hunk 1 above)
Comment (Norgus) [reply app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt]: I think I've matched the naming convention with my latest commit.

**Comment by dessalines** [comment]
It's alright, I'll fix the formatting issues shortly.

**Review by dessalines — APPROVED** [review]

**Comment by Norgus** [comment]
Thanks for adding my layout, and the help. I'll look forward to using it
when it reaches f-droid!

**Pull request merged**